apidoc/urlsession.yml

```diff
@@ -2,78 +2,79 @@
 name: Modules.URLSession
 summary: Wrapper to support iOS's NSURLSession class for background downloads.
 description: |
-    These APIs are supported on iOS 7 and later.
+  These APIs are supported on iOS 7 and later.
```

Comment:
This is incorrect. It has to be 4 spaces like all other docs. If the doc build succeeds with this setting, it has an issue in the generator.